check-diff.sh

```diff
@@ -554,6 +556,10 @@ function install_wp {
 
 	svn export -q "$SVN_URL" "$WP_CORE_DIR"
 
+	# Add workaround for running PHPUnit tests from source by touching built files.
+	mkdir -p $WP_CORE_DIR/src/wp-includes/css/dist/block-library/
```

Comment:
ShellCheck recommends wrapping all paths in double quotes to account for spaces, etc.

```
mkdir -p "$WP_CORE_DIR/src/wp-includes/css/dist/block-library"
```

Also the trailing slash can be omitted.